mythril/interfaces/cli.py

```diff
@@ -563,7 +563,7 @@ def set_config(args: Namespace):
     ):
         config.set_api_from_config_path()
 
-    if args.__dict__.get("address", None):
+    if args.__dict__.get("rpc", None):
```

Comment:
Why this change? We also need to establish an RPC connection to the default URL when there's no `--rpc` arg right?

Reply:
If rpc is not set then it takes the value from config at line 564. Without this we can't use testnets or ganache for queries when using solidity files.